### ISSUE TITLE
Can Not Add Custom Home Menu Items

### ISSUE THREAD

**Issue by WindowsAurora** [wyrm65/skin.apptv#29]
**Describe the bug**
I Can Not Add Custom Menu Items Going To Skin/Home Menu/Favorites And Actions Does Nothing The Add on Gets Downloaded But Clicking Actions Does Nothing No Menu Pulls Up Even Going To The Context Menu Add To Home Screen Isn't There
I'm using the latest github version

**To Reproduce**
1. Go to 'Settings
2. Click on 'Skin/Home
3. Scroll down to a empty slot and try to add a custom home menu via favourites
4. Nothing happens

**Expected behavior**
Clicking should do something

**Info**
 - OS: Android Tv
 - Version: 8.0

**Additional context**
this Also Happens In Linux flatpak

**Comment by rosekos** [comment]
yep its a bug with android sadly impossible to fix at this point
